xwiki-platform-core/xwiki-platform-ldap/xwiki-platform-ldap-authenticator/src/main/java/com/xpn/xwiki/plugin/ldap/XWikiLDAPSearchAttribute.java

```diff
@@ -35,9 +43,15 @@
     /**
      * Attribute value.
      */
+    @Deprecated
```

Comment:
I don't agree with @Deprecated, this field is still useful for the code expecting String. You could just indicate in the javadoc that it might be null.